datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -630,6 +635,30 @@ def _join_labels(self, metric, scraper_config):
                     if mapping_key in label_mapping and mapping_value in label_mapping[mapping_key]:
                         sample_labels.update(label_mapping[mapping_key][mapping_value])
 
+    def _ignore_metrics_by_label(self, scraper_config, metric_name, sample):
+        ignore_metrics_by_label = scraper_config['ignore_metrics_by_labels']
+        sample_labels = sample[self.SAMPLE_LABELS]
+        for label_key, label_values in ignore_metrics_by_label.items():
+            # Wildcard * means all metrics with label_key will be ignored
+            if label_values is not None and '*' in label_values:
+                self.log.debug("Detected wildcard for label %s", label_key)
+                label_values = None
+
+            if label_values is None:
+                if label_key in sample_labels:
+                    self.log.debug("Skipping metric %s due to label key matching: %s", metric_name, label_key)
+                    return True
+            elif len(label_values) == 0:
+                self.log.debug("Skipping filter label %s with an empty values list, did you mean to use '*' wildcard?", label_key)
```

Comment:
Should this be higher than `debug`?  If the code hits this, then its a misconfiguration which the user may not expect.